sdk/helper/certutil/helpers.go

```diff
@@ -887,9 +887,13 @@ func createCertificate(data *CreationBundle, randReader io.Reader, privateKeyGen
 		return nil, errutil.InternalError{Err: fmt.Sprintf("error getting subject key ID: %s", err)}
 	}
 
+	if data.Params.NotBeforeDuration > 0 && data.Params.NotBefore.IsZero() {
+		data.Params.NotBefore = time.Now().Add(-1 * data.Params.NotBeforeDuration)
+	}
```

Comment:
If I'm not mistaken, if `NotBefore.IsZero() && NotBeforeDuration == 0`, we end up with `NotBefore = 0`, unlike the current behavior of `NotBefore = -30s`? 

I'd probably prefer to keep existing behavior, and let the caller manually decide on a `NotBefore` that closely aligns with zero seconds for this case. In general, I'd expect there to be some amount of clock skew.

Reply:
You're right. I've adjusted this bit so that `NotBefore` defaults to `-30s`. 

If `not_before` is set, it takes precedence over `not_before_duration`.

Reply:
If I were to nitpick, I would ask for the opposite conditional ordering (`if !notBefore.isZero()`) to mirror the precedence, but I don't think it matters too much :-)